saleor/plugins/webhook/plugin.py

```diff
@@ -114,6 +115,51 @@ def category_deleted(self, category: "Category", previous_value: None) -> None:
                 payload, event_type, webhooks, category, self.requestor
             )
 
+    def gift_card_created(self, gift_card: "GiftCard", previous_value: None) -> None:
+        if not self.active:
+            return previous_value
+        event_type = WebhookEventAsyncType.GIFT_CARD_CREATED
+        if webhooks := get_webhooks_for_event(event_type):
+            payload = {"id": graphene.Node.to_global_id("GiftCard", gift_card.id)}
+            trigger_webhooks_async(
+                payload, event_type, webhooks, gift_card, self.requestor
+            )
+
+    def gift_card_updated(self, gift_card: "GiftCard", previous_value: None) -> None:
+        if not self.active:
+            return previous_value
+        event_type = WebhookEventAsyncType.GIFT_CARD_UPDATED
+        if webhooks := get_webhooks_for_event(event_type):
+            payload = {"id": graphene.Node.to_global_id("GiftCard", gift_card.id)}
+            trigger_webhooks_async(
+                payload, event_type, webhooks, gift_card, self.requestor
+            )
+
+    def gift_card_deleted(self, gift_card: "GiftCard", previous_value: None) -> None:
+        if not self.active:
+            return previous_value
+        event_type = WebhookEventAsyncType.GIFT_CARD_DELETED
+        if webhooks := get_webhooks_for_event(event_type):
+            payload = {"id": graphene.Node.to_global_id("GiftCard", gift_card.id)}
+            trigger_webhooks_async(
+                payload, event_type, webhooks, gift_card, self.requestor
+            )
+
+    def gift_card_status_changed(
+        self, gift_card: "GiftCard", previous_value: None
+    ) -> None:
+        if not self.active:
+            return previous_value
+        event_type = WebhookEventAsyncType.GIFT_CARD_STATUS_CHANGED
+        if webhooks := get_webhooks_for_event(event_type):
+            payload = {
+                "id": graphene.Node.to_global_id("GiftCard", gift_card.id),
+                "is_active": gift_card.is_active,
+            }
+            trigger_webhooks_async(
+                payload, event_type, webhooks, gift_card, self.requestor
+            )
```

Comment:
It seems like we can have a generic method for all gift card webhooks, as only `event_type` is changing.